src/tools/msemu_4to8_loader/makefile

```diff
@@ -4,7 +4,18 @@ CFLAGS = -mz80 --no-std-crt0 --code-loc 0x4000 --data-loc 0x4000
 AS = sdasz80
 ASFLAGS = -lso
 
-all: clean msemu_4to8_loader.bin
+all: clean msemu_4to8_loader.bin append_bin write_df
```

Comment:
Do we want to force a clean here?

Reply:
Yes, because this process relies on catting one binary to another. Its safer just to rebuild every time rather than letting users shoot themselves in the foot/have to think about what they are doing.